cmd/xamarin.go

```diff
@@ -72,121 +71,72 @@ and then hit Enter`
 	if err != nil {
 		return printXamarinScanFinishedWithError("Failed to analyze Xamarin solution: %s", err)
 	}
-	log.Debugf("xamSln: %#v", xamSln)
-
-	// filter only the iOS "app"" projects
-	xamarinProjectsToChooseFrom := []project.Model{}
-	for _, aXamarinProject := range xamSln.ProjectMap {
-		switch aXamarinProject.SDK {
-		case constants.SDKIOS, constants.SDKTvOS, constants.SDKMacOS:
-			if aXamarinProject.OutputType == "exe" {
-				// possible project
-				xamarinProjectsToChooseFrom = append(xamarinProjectsToChooseFrom, aXamarinProject)
-			}
-		default:
-			continue
+
+	if enableVerboseLog {
+		xamSlnJSON, err := json.MarshalIndent(xamSln, "", "\t")
```

Comment:
[Prefer `b` to `xamSlnJSON`](https://github.com/golang/go/wiki/CodeReviewComments#variable-names)